Cabal-syntax/src/Distribution/Compat/Lens.hs

```diff
@@ -1,86 +1,103 @@
 {-# LANGUAGE RankNTypes #-}
+
 -- | This module provides very basic lens functionality, without extra dependencies.
 --
 -- For the documentation of the combinators see <http://hackage.haskell.org/package/lens lens> package.
 -- This module uses the same vocabulary.
-module Distribution.Compat.Lens (
-    -- * Types
-    Lens,
-    Lens',
-    Traversal,
-    Traversal',
+module Distribution.Compat.Lens
+  ( -- * Types
+    Lens
+  , Lens'
+  , Traversal
+  , Traversal'
+
     -- ** LensLike
-    LensLike,
-    LensLike',
+  , LensLike
+  , LensLike'
+
     -- ** rank-1 types
-    Getting,
-    AGetter,
-    ASetter,
-    ALens,
-    ALens',
+  , Getting
+  , AGetter
+  , ASetter
+  , ALens
+  , ALens'
+
     -- * Getter
-    view,
-    use,
-    getting,
+  , view
+  , use
+  , getting
+
     -- * Setter
-    set,
-    over,
+  , set
+  , over
+
     -- * Fold
-    toDListOf,
-    toListOf,
-    toSetOf,
+  , toDListOf
+  , toListOf
+  , toSetOf
+
     -- * Lens
-    cloneLens,
-    aview,
+  , cloneLens
+  , aview
+
     -- * Common lenses
-    _1, _2,
+  , _1
+  , _2
+
     -- * Operators
-    (&),
-    (^.),
-    (.~), (?~), (%~),
-    (.=), (?=), (%=),
-    (^#),
-    (#~), (#%~),
+  , (&)
+  , (^.)
+  , (.~)
+  , (?~)
+  , (%~)
+  , (.=)
+  , (?=)
+  , (%=)
+  , (^#)
+  , (#~)
+  , (#%~)
+
     -- * Internal Comonads
-    Pretext (..),
+  , Pretext (..)
+
     -- * Cabal developer info
     -- $development
-    ) where
+  ) where
 
-import Prelude()
 import Distribution.Compat.Prelude
+import Prelude ()
 
 import Control.Monad.State.Class (MonadState (..), gets, modify)
 
-import qualified Distribution.Compat.DList as DList
 import qualified Data.Set as Set
+import qualified Distribution.Compat.DList as DList
 
 -------------------------------------------------------------------------------
 -- Types
 -------------------------------------------------------------------------------
 
-type LensLike  f s t a b = (a -> f b) -> s -> f t
-type LensLike' f s   a   = (a -> f a) -> s -> f s
+type LensLike f s t a b = (a -> f b) -> s -> f t
+type LensLike' f s a = (a -> f a) -> s -> f s
```

Comment:
More vertical alignment unhappiness... I'm leaving comments like this so that others could see where we are going.